### ISSUE TITLE
Fix ugly CSS Font loading for non-inlined font-face declaring CSS

### ISSUE THREAD

**Issue by Munter** [Munter/subfont#37]
Subfont can either inline the font-face declaring CSS into the HTML pages or not. IF you chose to keep the CSS external, the preload polyfill basically redeclares the font-face in the CSS font-loading API.

Currently it looks like this example from https://www.monbanquet.fr/:

```js
try {
    new FontFace('\'Titillium Web__subset\'', 'url(\'' + '/subfont/Titillium_Web-700-6aaf856d7c.woff2' + '\') format(\'woff2\'), url(\'' + '/subfont/Titillium_Web-700-602e1f37aa.woff' + '\') format(\'woff\')', {
        'font-weight': 700
    }).load();
    new FontFace('\'Titillium Web__subset\'', 'url(\'' + '/subfont/Titillium_Web-900-342ea6d442.woff2' + '\') format(\'woff2\'), url(\'' + '/subfont/Titillium_Web-900-672c83803f.woff' + '\') format(\'woff\')', {
        'font-weight': 900
    }).load();
    new FontFace('\'Titillium Web__subset\'', 'url(\'' + '/subfont/Titillium_Web-600-350a161aeb.woff2' + '\') format(\'woff2\'), url(\'' + '/subfont/Titillium_Web-600-9ab0628bf2.woff' + '\') format(\'woff\')', {
        'font-weight': 600
    }).load();
    new FontFace('\'Titillium Web__subset\'', 'url(\'' + '/subfont/Titillium_Web-400-f58727bc0c.woff2' + '\') format(\'woff2\'), url(\'' + '/subfont/Titillium_Web-400-0854463e37.woff' + '\') format(\'woff\')', {
        'font-weight': 400
    }).load();
    new FontFace('\'Titillium Web__subset\'', 'url(\'' + '/subfont/Titillium_Web-300-772d109a3d.woff2' + '\') format(\'woff2\'), url(\'' + '/subfont/Titillium_Web-300-1b38f7b3c9.woff' + '\') format(\'woff\')', {
        'font-weight': 300
    }).load()
} catch (e) {}
```

The above has been prettified to improve readability in the github paste. The original is just on one line.

All the string concatenation happening here could be greatly simplified to produce a readable output.

https://github.com/assetgraph/assetgraph/blob/master/lib/transforms/subsetFonts.js#L820-L826

**Comment by papandreou** [comment]
Yeah, it's because of the `'...'.toString('url')` having its function call omitted. Usually when we do that, we run `compressJavaScript` afterwards so that the string literals get constant folded, but subfont doesn't do that.

**Comment by papandreou** [comment]
I think the easiest fix would be to add a flag that tells `subsetFonts` not to add the `.toString('url')`s instead of removing and constant folding them afterwards.

**Comment by Munter** [comment]
For the assetgraph transform of like to keep the references alive here. So I do need the tostrings. I guess subfont should compress, since it is supposed to be run stand alone

**Comment by Munter** [comment]
Fixed in v3.6.2
